src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,8 +1,7 @@
 package core.mate.academy.model;
 
 /**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
+ * Add some custom fields that could be only in Excavator Do not remove no-args constructor
```

Comment:
The comment suggests adding custom fields specific to an Excavator. Consider adding fields such as `bucketCapacity`, `armLength`, or other Excavator-specific attributes.